### PR TITLE
update docs

### PR DESCRIPTION
**Pull request by bturkus**
updated documentation for most of the impt new stuff (but nothing I wrote is precious; very open to rewrites). 
also: plz squash my commits; sorry I suck at git/giuthub.

### CODE REVIEW

File: Resources/Documentation/hardware.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
 # Needed Hardware
 
-- a computer running a recent macOS (vrecord 2017-10-17 has been tested on macOS 10.10, 10.11, 10.12 and 10.13)
-- a Blackmagic card from:
+- a computer running a recent version of macOS or select linux distributions (vrecord 2019-01-25 has been tested on macOS 10.11, 10.12, 10.13, and 10.14, and Ubuntu 18.04)
```
Comment (privatezero) [inline Resources/Documentation/hardware.md]: Sorry for one more tiny thing! It has also been tested on Ubuntu 16.04!

**Comment by privatezero** [comment]
Looks great to me!! Pointed out just _onnnee_ more little thing (sorry!) Thank you so much @bturkus!

File: Resources/Documentation/settings.md [hunk 2]
```diff
@@ -6,13 +6,13 @@ Run edit mode by typing:
 ```
 vrecord -e
 ```
-After selecting all of your options and clicking "OK" the you will be prompted to enter a unique ID for the file. After the ID is entered, the incoming video signal will be recorded to a file with some associated metadata files. When you are done recording, close the vrecord window. If you've set a time limit for capture the vrecord window should automatically close when the time limit has been reached.
+After selecting all of your options and clicking "OK," you will be prompted to enter a unique ID for the file. After the ID is entered, the incoming video signal will be recorded to a file with some associated metadata files. When you are finished recording, you can close the vrecord window. If you've set a time limit for capture, the vrecord window will automatically close when the time limit has been reached.
```
Comment (retokromer) [inline Resources/Documentation/settings.md]: Is it `clicking "OK," you` in English? (In languages I know it would be `clicking "OK", you`.)

Comment (bturkus) [reply Resources/Documentation/settings.md]: yeah inside parenthetical (to my knowledge)

File: Resources/Documentation/hardware.md (hunk 1 above)
Comment (bturkus) [reply Resources/Documentation/hardware.md]: added!

**Pull request merged**